packages/vite/CHANGELOG.md

```diff
@@ -19,9 +19,9 @@
 Today, we're taking another big step in Vite's story. The Vite [team](/team), [contributors](https://github.com/vitejs/vite/graphs/contributors), and ecosystem partners are excited to announce the release of the next Vite major:
```

Comment:
The /team here isn't updated

Reply:
It's my fault. I opened this PR without checking it carefully 😭

Reply:
Made a PR here: https://github.com/vitejs/vite/pull/18804

Reply:
No worries! Yeah that's the link. You can open another pr if you'd like to.